scripts/rollup/build.js

```diff
@@ -408,6 +408,33 @@ function shouldSkipBundle(bundle, bundleType) {
   return false;
 }
 
+// Strip unused require() statements for pure externals modules from the bundle.
+// Rollup's treeshake option should do this, but it doesn't work.
+function stripNoSideEffectImports(bundleType, pureExternalModules, filePath) {
+  const isProduction = isProductionBundleType(bundleType);
+
+  // Dead code elimintation is only applied to production bundles.
+  if (!isProduction) {
+    return;
+  }
+
+  const codeIn = fs.readFileSync(filePath, 'utf-8');
+  let codeOut = codeIn;
+
+  pureExternalModules.forEach(module => {
+    const regExp = new RegExp(
+      `(?<!= *)require\\(["']${module}["']\\)[,;]`,
```

Comment:
What does the first part of this do?

Reply:
It's a negative look behind to make sure the required module isn't being assigned to a variable.

Reply:
To be clear, this is definitely a hack. ☹️ I'm just not sure how to strip the unused import any other way. And if it's important to us to avoid importing the 816 byte interaction-tracking production bundle.